.changeset/real-fishes-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/scaffolder-backend-module-aws': major
```

Comment:
```suggestion
'@roadiehq/scaffolder-backend-module-aws': minor
```
I think a minor change should be enough, this is an addition, we are not breaking the existing features.